cdc/sink/mysql_params.go

```diff
@@ -46,6 +46,7 @@ const (
 	defaultWriteTimeout        = "2m"
 	defaultDialTimeout         = "2m"
 	defaultSafeMode            = true
+	defaultRCTxnIsolation      = "READ-COMMITTED"
```

Comment:
```suggestion
	defaultTxnIsolation      = "READ-COMMITTED"
```

Reply:
Besides, this variable may be changed by no way.

Reply:
if so, I slightly prefer `defaultTxnIsolationRC`. rest lgtm